src/test/scala/com/snowplowanalytics/indicative/TransformationSpec.scala

```diff
@@ -30,15 +30,199 @@ class TransformationSpec extends Specification with ScalaCheck with Matchers {
   def is = s2"""
 
     integration test                                                $e1
-    integration test without user identifying property              $e2
-    should be transformed with real contexts generated from schemas $e3
-    flattenJson should work on empty arrays                         $e4
-    flattenJson should work on empty objects                        $e5
+    integration test matching app_id                                $e2
```

Comment:
Tried to keep the existing numbering? scheme in place here, but also keep the tests in a logical order.